### DISCUSSION TITLE
hope add limit residence shop

### DISCUSSION THREAD

**Discussion by 8MiYile** [QuickShop-Community/QuickShop-Hikari#1720] [Ideas]
limit residence shop
example:
i have 5 residences, but admin set only 1 residence, i'm created, but i'm goto another my residence tried create quickshop, because this limit is use, so i can't create

**Comment by YuanYuanOwO** [comment]
请使用 DeepL 转换成英语。
我不会开发

**Reply by 8MiYile** [reply]
okay i'm use deepl and input
